bigquery_etl/docs/bqetl/templates/commands.md

```diff
@@ -0,0 +1,40 @@
+{% macro command_docs(cmd, n) -%}
+{{ "#" * n }} `{{ cmd.name }}`
+
+{{ cmd.description }}
+
+{% if cmd.commands | length > 0 -%}
+{% for command in cmd.commands -%}
+    {{ command_docs(command, n + 1) }}
+{% endfor -%}
+{% else -%}
+**Usage**
```

Comment:
Not defining this as a section to prevent mkdocs from adding an entry to the TOC.